src/intrinsics.cr

```diff
@@ -1,6 +1,6 @@
 lib Intrinsics
   fun debugtrap = "llvm.debugtrap"
-  {% if flag?(:x86_64) %}
+  {% if flag?(:x86_64) || flag(:aarch64) %}
```

Comment:
I thought of adding the `:abi64` and `:abi32` flags, based on the architecture, to solve this.